ci/docker/Dockerfile.test.arm_qemu

```diff
@@ -0,0 +1,46 @@
+# -*- mode: dockerfile -*-
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Dockerfile to build and run MXNet on Ubuntu 16.04 for CPU
+
+FROM ubuntu:16.04
+
+WORKDIR /work
+
+RUN apt-get update
+COPY install/ubuntu_python3.sh /work/
+RUN /work/ubuntu_python3.sh
+RUN apt -y install python
```

Comment:
Why this is needed? Doesn't ubuntu_python3.sh do what needs to be done?

Reply:
it's needed for ansible

Reply:
Since all other installations are in scripts maybe you want to have a dedicated script for this then. And why you would want python 2 anyway?